### ISSUE TITLE
Performance gap with (Madry et al., 2018)

### ISSUE THREAD

**Issue by Harry-Up** [MadryLab/robust-features-code#9]
Thanks for your work and sharing! I am confused about the performance gap between this work and [(Madry et al., 2018)](https://arxiv.org/pdf/1706.06083.pdf). Their model robust accuracy is 47% compared with 42% in this paper under PGD attacks with $\epsilon=8/255$ and $l_{\infty}$ norm. I re-train a model by the Pytorch [repo](https://github.com/louis2889184/pytorch-adversarial-training) and obtain the similar results in the paper. Any suggestion about the performance gap? Thanks!
